patches/common/chromium/.patches.yaml

```diff
@@ -179,7 +179,7 @@ patches:
     Linux, which may cause issues in a context like Electron, where the main
     and renderer threads are supposed to keep inherited permissions over the
     system.
-    
+
```

Comment:
Thanks! )

Reply:
That was automatic from my Atom, I hope it doesn't break anything :D